telegrambots/src/main/java/org/telegram/telegrambots/bots/DefaultAbsSender.java

```diff
@@ -87,6 +100,29 @@ public final DefaultBotOptions getOptions() {
         return options;
     }
 
+    protected CloseableHttpClient createHttpClient() {
+        CloseableHttpClient localClient = null;
+
+        if (options.getCredentialsProvider() != null) {
```

Comment:
Why such order?

```
if(not null) {
} else (not not null) {
}
```
in my opinion is more difficult to understand than
```
if(null){
} else (not null) {
}
```

Reply:
As for me it does not matter.

Reply:
What if it was

```
if (not null) {}
else if (null) {}
```